pkg/app/app.go

```diff
@@ -164,9 +165,36 @@ func (a *App) Start() {
 		a.log.Notice("Broker successfully bootstrapped on startup")
 	}
 
+	interval, err := time.ParseDuration(a.config.Broker.RefreshInterval)
+	a.log.Debug("RefreshInterval: %v", interval.String())
+	if err != nil {
+		a.log.Error(err.Error())
+		a.log.Error("Not using a refresh interval")
+	} else {
+		ticker := time.NewTicker(interval)
+		ctx, cancelFunc := context.WithCancel(context.Background())
+		defer cancelFunc()
+		go func() {
+			for {
+				select {
+				case v := <-ticker.C:
+					a.log.Info("Broker configured to refresh specs every %v seconds", interval)
+					a.log.Info("Attempting bootstrap at %v", v.UTC())
+					if _, err := a.broker.Bootstrap(); err != nil {
```

Comment:
Will we have a problem with constant bootstraps? So far bootstrap got called at startup which means no active provisions, etc. We would also run it when we called the endpoint which usually meant we probably didn't have any active provisions, etc. Will this auto bootstraps potentially cause problems?